src/hsmm_mvpy/models.py

```diff
@@ -108,7 +108,8 @@ def __init__(self, data, epoch_data=None, sfreq=None, cpus=1, event_width=50, sh
         self.event_width_samples = int(np.round(self.event_width / self.steps))
         if location is None:
             self.location = int(self.event_width / self.steps//2)+1
-        else: self.location =  int(np.rint(location / self.steps))
+        else:
```

Comment:
Indeed but the int(np.rint() insured that it was an int, perhaps then just add an assert?

Reply:
sure, or leave the `int(np.rint())` and just remove `/ self.steps` (which made the location go to 0 and it estimated overlapping events).

Do you have a preference for` int(np.rint())` or assert?

Reply:
We can just keep ```int(np.rint())``` I would say